pilot/pkg/model/authentication.go

```diff
@@ -97,13 +110,75 @@ func (policy *AuthenticationPolicies) addRequestAuthentication(configs []Config)
 	}
 }
 
+func apiModeToMutualTLSMode(mode v1beta1.PeerAuthentication_MutualTLS_Mode) MutualTLSMode {
```

Comment:
this is duplication with policy applier's `getMutualTlsmode` right?

Reply:
yeah :). Need some refactoring, may be when we deprecate alpha.

Reply:
Actually, they are same

Reply:
ok. leave a todo on the top of func then.

Reply:
Done